presto-memory/src/main/java/com/facebook/presto/plugin/memory/MemoryMetadata.java

```diff
@@ -176,16 +179,20 @@ public synchronized MemoryOutputTableHandle beginCreateTable(ConnectorSession se
         MemoryTableHandle table = new MemoryTableHandle(
                 connectorId,
                 nextId,
-                tableMetadata,
```

Comment:
Yes, I have added `assertTableNotExists` method which is called at the top of this method.